### DISCUSSION TITLE
aux_connected_status not responding

### DISCUSSION THREAD

**Discussion by adnovea** [slipx06/sunsynk-power-flow-card#263] [Q&A] [answered]
Is it me or the sensor "**aux_connected_status**" does not switch the color between "_aux_colour_" and "_aux_off_colour_" depending of the value "_on/off_" or "_0/1_" ?

I also tried to manually change the value of  "**aux_connected_status**" using the development without success.
Maybe there are exclusive sensors that prevent the switching ?

**Comment by slipx06** [comment, marked as answer]
It only works if there are no additional aux loads and it changes the colour of the aux icon.

`aux_connected_status = 0`
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/336cb636-de42-4171-ad3e-d117a46011b3)
`aux_connected_status = 1`
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/89549923-7b05-4e50-97ba-e3138fbdb0e7)

**Reply by adnovea** [reply]
Thanks for the explanation.
It works as you described it above.

Suggestion : May be in the future it would be possible to change the color when additional aux loads are set.
